client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -64,32 +70,43 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             error={touched.email && Boolean(errors.email)}
             value={values.email}
             onChange={handleChange}
+            variant="outlined"
           />
+          <InputLabel htmlFor="password">
+            <Typography className={classes.label} color="textPrimary">
+              password
+            </Typography>
+          </InputLabel>
           <TextField
             id="password"
-            label={<Typography className={classes.label}>Password</Typography>}
+            placeholder="Password"
             fullWidth
             margin="normal"
             InputLabelProps={{
               shrink: true,
             }}
             InputProps={{
               classes: { input: classes.inputs },
-              endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
             }}
             type="password"
             autoComplete="current-password"
             helperText={touched.password ? errors.password : ''}
             error={touched.password && Boolean(errors.password)}
             value={values.password}
             onChange={handleChange}
+            variant="outlined"
           />
           <Box textAlign="center">
-            <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Login'}
+            <Button type="submit" size="large" variant="contained" color="secondary" className={classes.submit}>
+              {isSubmitting ? <CircularProgress className={classes.circleColor} /> : 'Login'}
             </Button>
           </Box>
-          <div style={{ height: 95 }} />
+          <Box mt={3} textAlign="center" alignItems="center" justifyContent="center">
+            <Typography className={classes.boldText}> Don&apos;t have account? </Typography>
```

Comment:
```suggestion
            <Typography className={classes.boldText}> {"Don't have account? "}</Typography>
```

I think a lot simpler to use the Unicode character within a string instead of HTML entities 😄

Reply:
Haha! True, I will add that! 😀

![image](https://user-images.githubusercontent.com/60498899/138378030-81168be1-fef4-498b-929f-97f3cc5abe6e.png)

Reply:
This is hilarious 🤣